lib/AssemblyUtil/AssemblyUtilImpl.py

```diff
@@ -71,6 +72,39 @@ def get_assembly_as_fasta(self, ctx, params):
         # return the results
         return [file]
 
+    def get_fastas(self, ctx, ref_lst):
+        """
+        Given a reference list of KBase objects constructs a local Fasta file with the sequence data for each ref.
+        :param ref_lst: instance of type "KBaseOjbReferences" (Structure for
+           get_fasta function input and output: ref_lst - is an array of
+           KBase object references, such as KBaseGenomes.Genome,
+           KBaseSets.AssemblySet, etc. FASTA - is an array of fasta file
+           paths) -> list of String
+        :returns: instance of type "GetFASTAOutput" -> structure: parameter
+           "FASTA" instance of Dictionary.
+        """
+        # ctx is the context object
+        # return variables are: output
+        #BEGIN get_fastas
+
+        # Param check
+        if not ref_lst:
+            raise ValueError("Must provide reference list.")
```

Comment:
You could also check that the entries in the list look like valid refs, but the workspace will do that if not. usually better to fail early, but up to you.